src/sessions/video-session-handler.ts

```diff
@@ -635,7 +635,22 @@ export default class VideoSessionHandler extends BaseSessionHandler {
 
     if (session._resurrectVideoOnScreenShareEnd) {
       this.log('info', 'Restarting video track', { conversationId: session.conversationId, sessionId: session.id, sessionType: session.sessionType });
-      await this.setVideoMute(session, { conversationId: session.conversationId, mute: false }, true);
+      try {
+        await this.setVideoMute(session, { conversationId: session.conversationId, mute: false }, true);
+      } catch (err) {
```

Comment:
I'm not opposed but I'm wondering in a case where for some reason we actually aren't able to stop sharing our screen?  I mean I guess it could still be good for it to appear to the user they are not sharing their screen but would that mess things up on the backend?  And you mean instead of throwing the error again?  Yeah I wasn't entirely sure what to do there because I believe we react to the error in Volt which does just print it out though.  So I guess we can just skip that step?

Reply:
yeah so I think we should remove the `if` condition, and always `setVideoMute` to true and make sure the screen share is stopped.

Reply:
I pushed up the requested changes.  Though I didn't log the error in the catch block because we actually have a few places that are already logging the error.